src/utils/sqs.ts

```diff
@@ -0,0 +1,33 @@
+import { SqsService } from "./types";
+import {
+  SendMessageCommand, SendMessageCommandOutput,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { logger } from "./logger";
+
+export function sqsService(): SqsService {
```

Comment:
I think we could probably follow the same pattern as all the other services? Have a look at snsService, basically we abstract all the aws config to aws.ts